packages/pipeline-editor/src/PipelineEditorWidget.tsx

```diff
@@ -1049,6 +1052,9 @@ export class PipelineEditor extends React.Component<
     }
 
     const runtimes = await PipelineService.getRuntimes();
+    const local_runtime: any = { name: 'local', display_name: 'local' };
```

Comment:
I'm not sure `local` is the best value for `display_name`, I feel like we should figure out something better.

Based on text in the results dialog perhaps `Run in-place locally` would be a good value